src/test/java/com/synopsys/integration/alert/channel/rest/RestDistributionChannelTest.java

```diff
@@ -44,16 +44,16 @@
 import com.synopsys.integration.rest.request.Request;
 
 public class RestDistributionChannelTest extends ChannelTest {
-    @Test
+    // TODO figure out what this was supposed to be testing:
```

Comment:
This was test was to increase the test coverage of RestDistributionChannel to 100%. In hindsight testing the other channels like HipChat and Slack which extend RestDistributionChannel should be modified to get us to 100% code coverage for this class as well.  This test we may be able to delete.